pkg/planner/property/physical_property.go

```diff
@@ -297,6 +305,31 @@ func (p *PhysicalProperty) IsSubsetOf(keys []*MPPPartitionColumn) []int {
 	return matches
 }
 
+// NeedEnforceExchangerWithHashByEquivalence checks if the keys can match the needs of partition with equivalence.
+func (p *PhysicalProperty) NeedEnforceExchangerWithHashByEquivalence(keys []*MPPPartitionColumn) bool {
+	// keys is the HashCol. If the partition cols are a subset of the hash cols, then need to enforce exchange.
+	if len(p.MPPPartitionCols) < len(keys) {
```

Comment:
Please add the case "when the keys is more than 1" to check the correctness of the FD.ClosureOfStrict

Reply:
Constructing this under certain constraints is quite challenging. Moreover, regarding the tests for Functional Dependencies, they actually already exist, and they also include the equivalence of multiple columns.